### PR TITLE
[Python API] Update doc style

### PR DESCRIPTION
**Pull request by akuporos**
### Details:
 - *item1*
 - *...*

### Tickets:
 - 80658

### CODE REVIEW

File: src/bindings/python/src/pyopenvino/core/variable_state.cpp [hunk 1]
```diff
@@ -20,40 +20,22 @@ void regclass_VariableState(py::module m) {
...
         to a value specified as default for according node.
 
-        Parameters
-        ----------
-        None
-
-        Returns
-        ----------
-        reset : None
+        :rtype: None
```
Comment (jiwaszki) [inline src/bindings/python/src/pyopenvino/core/variable_state.cpp:23]:
```suggestion
```
None is just a void -- no need to mark it.

**Review by jiwaszki — CHANGES_REQUESTED** [review]
Overall most of the changes are requested to keep consistency. In some places there are functions without docstrings at all -- is there a plan to document them as well? (Example `convert_color` in PreProcessSteps)

**Comment by ilya-lavrenov** [comment]
is it a target for 2022.1?

File: src/bindings/python/src/pyopenvino/frontend/frontend.cpp [hunk 2]
```diff
@@ -46,32 +41,22 @@ void regclass_frontend_FrontEnd(py::module m) {
...
-                Parameters
-                ----------
-                function : Model
-                    Partially converted nGraph function.
-
-                Returns
-                ----------
-                convert : Model
-                    Fully converted nGraph function.
+                :param model: Partially converted nGraph function.
```
[the remark this answers is not in the document]
Comment (akuporos) [reply src/bindings/python/src/pyopenvino/frontend/frontend.cpp]: done

File: src/bindings/python/src/pyopenvino/frontend/frontend.cpp [hunk 3]
```diff
@@ -130,10 +103,8 @@ void regclass_frontend_FrontEnd(py::module m) {
                 Gets name of this FrontEnd. Can be used by clients
                 if frontend is selected automatically by FrontEndManager::load_by_model.
 
-                Parameters
-                ----------
-                get_name : str
-                    Current frontend name. Empty string if not implemented.
+                :return: Current frontend name. Empty string if not implemented.
```
[the remark this answers is not in the document]
Comment (akuporos) [reply src/bindings/python/src/pyopenvino/frontend/frontend.cpp]: done

File: src/bindings/python/src/pyopenvino/frontend/input_model.cpp [hunk 4]
```diff
@@ -241,10 +194,8 @@ void regclass_frontend_InputModel(py::module m) {
                 Removes any sinks directly attached to this place with all inbound data flow
                 if it is not required by any other output.
 
-                Parameters
-                ----------
-                place : Place
-                    Model place
+                :param place: Model place
```
[the remark this answers is not in the document]
Comment (akuporos) [reply src/bindings/python/src/pyopenvino/frontend/input_model.cpp]: done

File: src/bindings/python/src/pyopenvino/frontend/input_model.cpp [hunk 5]
```diff
@@ -272,37 +220,28 @@ void regclass_frontend_InputModel(py::module m) {
...
-                Parameters
-                ----------
-                place : Place
-                    Model place
-
-                Returns
-                ----------
-                get_partial_shape : PartialShape
-                    Partial shape for this place.
+                :param place: Model place
```
[the remark this answers is not in the document]
Comment (akuporos) [reply src/bindings/python/src/pyopenvino/frontend/input_model.cpp]: done

File: src/bindings/python/src/pyopenvino/graph/preprocess/pre_post_process.cpp [hunk 6]
```diff
@@ -75,15 +65,10 @@ static void regclass_graph_PreProcessSteps(py::module m) {
...
-            Parameters
-            ----------
-            value : float
-                Value to divide.
-
-            Returns
-            ----------
-            scale : PreProcessSteps
-                Reference to itself to allow chaining of calls in client's code in a builder-like manner.
+            :param value: Value to divide.
```
[the remark this answers is not in the document]
Comment (akuporos) [reply src/bindings/python/src/pyopenvino/graph/preprocess/pre_post_process.cpp]: done

File: src/bindings/python/src/pyopenvino/graph/preprocess/pre_post_process.cpp [hunk 7]
```diff
@@ -96,15 +81,10 @@ static void regclass_graph_PreProcessSteps(py::module m) {
...
-            Parameters
-            ----------
-            value : List[float]
-                Value to divide.
-
-            Returns
-            ----------
-            scale : PreProcessSteps
-                Reference to itself to allow chaining of calls in client's code in a builder-like manner.
+            :param values: Value to divide.
```
[the remark this answers is not in the document]
Comment (akuporos) [reply src/bindings/python/src/pyopenvino/graph/preprocess/pre_post_process.cpp]: done

File: src/bindings/python/src/pyopenvino/graph/preprocess/pre_post_process.cpp [hunk 8]
```diff
@@ -242,14 +208,10 @@ static void regclass_graph_PostProcessSteps(py::module m) {
...
-            operation : function taking Output<Node> as input argument and returning Output<Node> after postprocessing.
-
-            Returns
-            ----------
-            custom : PostProcessSteps
-                Reference to itself to allow chaining of calls in client's code in a builder-like manner.
+            :param operation: function taking Output<Node> as input argument and returning Output<Node> after postprocessing.
+            :type operation: openvino.runtime.Output
+            :return: Reference to itself to allow chaining of calls in client's code in a builder-like manner.
+            :rtype: PostProcessSteps
```
[the remark this answers is not in the document]
Comment (akuporos) [reply src/bindings/python/src/pyopenvino/graph/preprocess/pre_post_process.cpp]: done

File: src/bindings/python/src/pyopenvino/graph/preprocess/pre_post_process.cpp [hunk 9]
```diff
@@ -137,14 +112,10 @@ static void regclass_graph_PreProcessSteps(py::module m) {
...
-            operation : function taking Output<Node> as input argument and returning Output<Node> after preprocessing.
-
-            Returns
-            ----------
-            custom : PreProcessSteps
-                Reference to itself to allow chaining of calls in client's code in a builder-like manner.
+            :param operation: function taking Output<Node> as input argument and returning Output<Node> after preprocessing.
+            :type operation: openvino.runtime.Output
+            :return: Reference to itself to allow chaining of calls in client's code in a builder-like manner.
+            :rtype: openvino.runtime.preprocess.PreProcessSteps
```
[the remark this answers is not in the document]
Comment (akuporos) [reply src/bindings/python/src/pyopenvino/graph/preprocess/pre_post_process.cpp]: done

File: src/bindings/python/src/pyopenvino/graph/preprocess/pre_post_process.cpp [hunk 10]
```diff
@@ -269,15 +231,10 @@ static void regclass_graph_InputTensorInfo(py::module m) {
...
-            type : Type
-                Client's input tensor element type.
-
-            Returns
-            ----------
-            tensor : InputTensorInfo
-                Reference to itself to allow chaining of calls in client's code in a builder-like manner.
+            :param type: Client's input tensor element type.
+            :type type: openvino.runtime.Type
+            :return: Reference to itself to allow chaining of calls in client's code in a builder-like manner.
```
[the remark this answers is not in the document]
Comment (akuporos) [reply src/bindings/python/src/pyopenvino/graph/preprocess/pre_post_process.cpp]: done

File: src/bindings/python/src/pyopenvino/graph/passes/manager.cpp [hunk 11]
```diff
@@ -121,20 +119,19 @@ void regclass_graph_passes_Manager(py::module m) {
...
-        version : str
-            sets the version of the IR which will be generated.
+
+        :param pass_name: string to set the type of a pass
+        :type pass_name: str
+        :param xml_path: path where .xml file will be saved
+        :type xml_path: str
+        :param bin_path: path where .bin file will be saved
+        :type bin_path: str
+        :param version: sets the version of the IR which will be generated.
```
[the remark this answers is not in the document]
Comment (akuporos) [reply src/bindings/python/src/pyopenvino/graph/passes/manager.cpp]: done

File: src/bindings/python/src/pyopenvino/graph/preprocess/pre_post_process.cpp [hunk 12]
```diff
@@ -54,15 +49,10 @@ static void regclass_graph_PreProcessSteps(py::module m) {
...
-            Parameters
-            ----------
-            values : List[float]
-                Values to subtract.
-
-            Returns
-            ----------
-            selfan : PreProcessSteps
-                Reference to itself to allow chaining of calls in client's code in a builder-like manner.
+            :param values: Value to subtract.
```
[the remark this answers is not in the document]
Comment (akuporos) [reply src/bindings/python/src/pyopenvino/graph/preprocess/pre_post_process.cpp]: done

File: src/bindings/python/src/pyopenvino/graph/passes/manager.cpp [hunk 13]
```diff
@@ -90,18 +89,17 @@ void regclass_graph_passes_Manager(py::module m) {
...
+
+        :param pass_name: string to set the type of a pass
+        :type pass_name: str
+        :param output_files: tuple which contains paths where .xml and .bin files will be saved
+        :type output_files: Tuple[str, str]
+        :param version: sets the version of the IR which will be generated.
             Supported versions are:
                             - "UNSPECIFIED" (default) : Use the latest or function version
                             - "IR_V10" : v10 IR
                             - "IR_V11" : v11 IR
```
[the remark this answers is not in the document]
Comment (akuporos) [reply src/bindings/python/src/pyopenvino/graph/passes/manager.cpp]: done

File: src/bindings/python/src/pyopenvino/graph/passes/manager.cpp [hunk 14]
```diff
@@ -76,10 +76,9 @@ void regclass_graph_passes_Manager(py::module m) {
                 py::arg("pass_name"),
                 R"(
         Set the type of register pass for pass manager.
-        Parameters
-        ----------
-        pass_name : str
-            string to set the type of a pass
+
+        :param pass_name : string to set the type of a pass
```
[the remark this answers is not in the document]
Comment (akuporos) [reply src/bindings/python/src/pyopenvino/graph/passes/manager.cpp]: done

File: src/bindings/python/src/pyopenvino/frontend/place.cpp [hunk 15]
```diff
@@ -21,33 +21,27 @@ void regclass_frontend_Place(py::module m) {
...
               &ov::frontend::Place::get_names,
               R"(
                 All associated names (synonyms) that identify this place in the graph in a framework specific way.
 
-                Returns
-                ----------
-                get_names : List[str]
-                    A vector of strings each representing a name that identifies this place in the graph.
+                :return: A vector of strings each representing a name that identifies this place in the graph.
                     Can be empty if there are no names associated with this place or name cannot be attached.
```
[the remark this answers is not in the document]
Comment (akuporos) [reply src/bindings/python/src/pyopenvino/frontend/place.cpp]: done

File: src/bindings/python/src/pyopenvino/frontend/manager.cpp [hunk 16]
```diff
@@ -66,15 +59,10 @@ void regclass_frontend_FrontEndManager(py::module m) {
...
-                model_path : str
-                    Path to model file/directory.
-
-                Returns
-                ----------
-                load_by_model : FrontEnd
-                    Frontend interface for further loading of models. 'None' if no suitable frontend is found
+                :param model_path: Path to model file/directory.
+                :type model_path: str
+                :return: Frontend interface for further loading of models. 'None' if no suitable frontend is found
```
[the remark this answers is not in the document]
Comment (akuporos) [reply src/bindings/python/src/pyopenvino/frontend/manager.cpp]: done

File: src/bindings/python/src/pyopenvino/frontend/manager.cpp [hunk 17]
```diff
@@ -66,15 +59,10 @@ void regclass_frontend_FrontEndManager(py::module m) {
...
-                Parameters
-                ----------
-                model_path : str
-                    Path to model file/directory.
-
-                Returns
-                ----------
-                load_by_model : FrontEnd
-                    Frontend interface for further loading of models. 'None' if no suitable frontend is found
+                :param model_path: Path to model file/directory.
```
[the remark this answers is not in the document]
Comment (akuporos) [reply src/bindings/python/src/pyopenvino/frontend/manager.cpp]: done

File: src/bindings/python/src/pyopenvino/frontend/input_model.cpp [hunk 18]
```diff
@@ -375,12 +304,9 @@ void regclass_frontend_InputModel(py::module m) {
...
-            ----------
-            place : Place
-                Model place.
-
-            value : ndarray
-                New value to assign.
+            :param place: Model place.
+            :type place: openvino.frontend.Place
+            :param value: New value to assign.
+            :type value: ndarray
```
[the remark this answers is not in the document]
Comment (akuporos) [reply src/bindings/python/src/pyopenvino/frontend/input_model.cpp]: done

File: src/bindings/python/src/pyopenvino/frontend/input_model.cpp [hunk 19]
```diff
@@ -272,37 +220,28 @@ void regclass_frontend_InputModel(py::module m) {
...
     im.def("get_outputs",
            &ov::frontend::InputModel::get_outputs,
            R"(
                 Returns all outputs for a model. An output is a terminal place in a graph where data escapes the flow.
 
-                Returns
-                ----------
-                get_outputs : List[Place]
-                    A list of output places
+                :return: A list of output places
```
[the remark this answers is not in the document]
Comment (akuporos) [reply src/bindings/python/src/pyopenvino/frontend/input_model.cpp]: done

File: src/bindings/python/src/pyopenvino/frontend/input_model.cpp [hunk 20]
```diff
@@ -272,37 +220,28 @@ void regclass_frontend_InputModel(py::module m) {
...
     im.def("get_inputs",
            &ov::frontend::InputModel::get_inputs,
            R"(
                 Returns all inputs for a model.
 
-                Returns
-                ----------
-                get_inputs : List[Place]
-                    A list of input places.
+                :return: A list of input places
```
[the remark this answers is not in the document]
Comment (akuporos) [reply src/bindings/python/src/pyopenvino/frontend/input_model.cpp]: done

**Pull request merged**